frontend/src/utils/d3Utils.js

```diff
@@ -0,0 +1,180 @@
+import * as d3 from 'd3';
```

Comment:
We have utils repeated in this file name. If there's no way to categorize these utils beyond "d3" maybe they could live in the index file.

Reply:
Deleted this extra file in b7ffb6d03a940b75d2f88f941ff8b25e8effee72